SyncDataServices/Http/HttpShipmentInternalDataClient.cs

```diff
@@ -2,44 +2,122 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Net.Http;
+using System.Net.Http.Headers;
 using System.Text;
 using System.Text.Json;
 using System.Threading.Tasks;
 using DiAnterExpress.Dtos;
 using DiAnterExpress.Models;
 using GraphQL;
 using GraphQL.Client.Abstractions;
+using GraphQL.Client.Http;
 using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
 
 namespace DiAnterExpress.SyncDataServices.Http
 {
     public class HttpShipmentInternalDataClient : IShipmentInternalDataClient
     {
-        private readonly IGraphQLClient _client;
-        private readonly IConfiguration _configuration;
-
-        public HttpShipmentInternalDataClient(IGraphQLClient client)
+        private readonly GraphQLHttpClient _client;
+        public HttpShipmentInternalDataClient(GraphQLHttpClient client)
         {
             _client = client;
         }
```

Comment:
Related to https://github.com/agit-bootcamp-2022/DiAnterExpress/pull/20/files#r794659107

Dipindah ke constructornya, biar ga tabrakan nanti sama DataClient untuk Tokopodia
```suggestion
        private GraphQLHttpClient _client;
        private IConfiguration _configuration;
        private GraphQLHttpClientOptions graphqlOptions;

        public TokopodiaDataClient(IConfiguration configuration)
        {
            _configuration = configuration;

            graphqlOptions = new GraphQLHttpClientOptions
            {
                EndPoint = new Uri(_configuration["UangTransURI"], UriKind.Absolute)
            };

            _client = new GraphQLHttpClient(
                graphqlOptions,
                new NewtonsoftJsonSerializer()
            );
        }
```

Reply:
setelah aku cek lagi, kayaknya gaperlu bikin `new GraphQLHttpClientOptions`, langsung aja masukkin URI nya ke `new GraphQLHttpClient`

```c#
        private GraphQLHttpClient _client;
        private IConfiguration _configuration;
        public TokopodiaDataClient(IConfiguration configuration)
        {
            _configuration = configuration;

            _client = new GraphQLHttpClient(
                _configuration["UangTransURI"],
                new NewtonsoftJsonSerializer()
            );
        }
```

Reply:
berry nice